specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/definitions.json

```diff
@@ -75,6 +129,51 @@
       },
       "description": "Tenant access information update parameters of the API Management service"
     },
+    "TenantSettingsCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TenantSettingsContract"
+          },
+          "description": "Page values.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any.",
+          "readOnly": true
+        }
+      },
+      "description": "Paged AccessInformation list representation."
+    },
+    "TenantSettingsContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/TenantSettingsContractProperties",
+          "description": "TenantSettings entity contract properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Tenant Settings."
+    },
+    "TenantSettingsContractProperties": {
+      "properties": {
+        "settings": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
```

Comment:
Please explain how this object will look and what possible properties would be there.

Reply:
Hi @chiragg4u ,
This type description indicates it's a Dictionalry<string, string> and will not allow any other types. (https://swagger.io/docs/specification/data-models/dictionaries/ )